frontend/components/Comment.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import Link from 'next/link'
+import Form from '../components/Form'
+import { Consumer } from '../Context'
+class Comment extends Component {
+  state = { editValue: '', errors: [], textfieldOpen: false }
+  change = event => {
+    this.setState({ editValue: event.target.value })
+  }
+  componentDidMount() {
+    this.setState(prevState => ({
+      editValue: this.props.content
+    }))
+  }
```

Comment:
Putting props into state is an anti pattern in one way data flow scenarios because you don't have a single source of truth for your data. You are duplicating it in two places. You will have to keep to two places in sync if they both use `content` in this scenario. This is pretty advanced so just keep it in the back of your mind but its okay to ignore for now. Read more here https://overreacted.io/writing-resilient-components/#principle-1-dont-stop-the-data-flow

Reply:
Will keep in mind, cheers